tsconfig.compodoc.json

```diff
@@ -476,6 +476,9 @@
       "@spartacus/cdc/root": [
         "integration-libs/cdc/root/public_api"
       ],
+      "@spartacus/cdc/manage-users": [
```

Comment:
please move this to the last of this array.

Reply:
done